From 90c1bb8a1df8fd1d84fafaf10b135b0f2b55c681 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Mon, 10 Feb 2020 16:06:54 +0200
Subject: [PATCH] [SW-0] goya: pldm: pll: remove hl_pll_is_configurable()

The Goya PLDM implementation of hl_pll_is_configurable() function is
just verifies that the only PLL configured is the PCI one, but that is
the only PLL defined for Goya PLDM.
Remove the hl_pll_is_configurable() Goya PLDM implementation as the
default (weak) one does the job as well.

Change-Id: I18890675a7a210856fdfdcdab2e0b2c32e7ccd0d
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/goya/pldm.c b/goya/pldm.c
index 2cbbc97..5ddd4b1 100644
--- a/goya/pldm.c
+++ b/goya/pldm.c
@@ -28,11 +28,6 @@
 	},
 };
 
-bool hl_pll_is_configurable(struct hl_pll_cfg *cfg)
-{
-	return cfg->base == mmPSOC_PCI_PLL_BASE;
-}
-
 int hl_pll_get_cfg(struct hl_pll_cfg **hl_pll_cfg)
 {
 	int pll_num = ARRAY_SIZE(goya_pll_cfg);
